modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/common/edit_kb_article.jsp

```diff
@@ -59,16 +61,44 @@ if (editKBArticleDisplayContext.isPortletTitleBasedNavigation()) {
 
 						<c:choose>
 							<c:when test='<%= FeatureFlagManagerUtil.isEnabled("LPS-188060") %>'>
-								<clay:dropdown-menu
-									cssClass="c-mr-3"
-									displayType="primary"
-									dropdownItems="<%= editKBArticleDisplayContext.getEditKBArticleActionDropdownItems() %>"
-									icon="caret-bottom"
-									id='<%= liferayPortletResponse.getNamespace() + "publishDropdown" %>'
-									label="<%= editKBArticleDisplayContext.getPublishButtonLabel() %>"
-									name="publishDropdown"
-									small="<%= true %>"
-								/>
+								<c:choose>
+									<c:when test="<%= kbArticle.isScheduled() %>">
+
```

Comment:
Use instead a new method in DisplayContext called `isScheduled()` similar to the existing `isPending()`, which returns false if kbArticle == null.